repository/src/main/java/ca/bc/gov/repository/bcsc/BcscAuthRepo.kt

```diff
@@ -36,11 +28,11 @@ class BcscAuthRepo(
     private val patientLocalDataSource: PatientLocalDataSource,
 ) {
 
-    private lateinit var authState: AuthState
-    private lateinit var authService: AuthorizationService
-    private lateinit var authServiceConfiguration: AuthorizationServiceConfiguration
+    private lateinit var authState: net.openid.appauth.AuthState
```

Comment:
Is it possible to declare the imports in the top for  AuthState, AuthorizationService, AuthorizationServiceConfiguration, etc?

Reply:
updated, auto import cause this issue